changelog.md

```diff
@@ -66,6 +69,7 @@
 - Fix crash in bridge when calling `vmq-admin session show` by fixing the
   `vmq_ql` row initializer to handle plugin sessions (the bridge starts a local
   plugin session).
+>>>>>>> 93838e09778c8467768b4e5f33520d57740a03c8
```

Comment:
Hey @codeadict - seems something went wrong in the last commit :D

Reply:
Ohh wow sorry about that one, fixing

Reply:
no worries - can you put the new log line at the bottom of the list? makes it easier when cherry picking for bugfix-releases.

Reply:
Yup done @larshesel